apps/frontend/src/services/neighborhoods.ts

```diff
@@ -28,6 +28,30 @@ async function getSingleNeighborhood(
   return null;
 }
 
+async function deleteNeighborhood(
+  id: number,
+): Promise<Response | { success: string } | { error: string }> {
+  const user = getStoredUser();
+  if (!user) return redirect('/login');
+
+  const headers = { authorization: `Bearer ${user.token}` };
+  await axios.delete(`${BASE_URL}/${id}`, { headers });
+    
+  return redirect('/')
+}
+
+async function createNeighborhood(
+  neighborhoodData: EditNeighborhoodData,
```

Comment:
It might be more intuitive, even though they're equivalent, to use the `CreateNeighborhoodData` type from the backend here and in the HomePage.

Reply:
They are not equivalent. `CreateNeighborhodData` has the following properties 'admin_id', 'name' and 'description, while `EditNeighborhoodData` has only `name` and `description`.
When creating a neighborhood on the frontend we don't need the admin_id. That id is added when the request reaches the backend.  
<img width="1056" alt="Screenshot 2023-11-27 at 19 39 11" src="https://github.com/neighborhood-app/app/assets/40914437/09c4cab0-38ba-4e23-88a7-a6c7135141af">
You can see it on line 79.
The better option would be to rename `EditNeighborhoodData` to something more general.

Reply:
Ah, I see! We could just assign `EditNeighborhoodData` to a `CreateNeighborhoodData` type on the frontend.